src/index.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+import './style.css';
+
+function showTodo(todo) {
+    const list = document.querySelector('.list');
+    const newlist = document.createElement('li');
+    newlist.classList.add('newlist');
+    newlist.innerHTML = `
+      <input id="${todo.index}" class="check" type="checkbox"/>
+      <span class="text">${todo.description}</span>
+      <button class="delete"><img src="https://e7.pngegg.com/pngimages/179/938/png-clipart-computer-icons-hamburger-button-dots-kebab-menu-text-rectangle-thumbnail.png"></button>
+    `;
+    list.append(newlist);
+}
+
+const todoList = [];
+let id = 1;
+
+function add(text) {
+    const todo = {
+        description: text,
+        completed: false,
+        index: id += 1,
+    };
+    todoList.push(todo);
+    showTodo(todo);
+}
+
+const form = document.querySelector('.main');
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const input = document.querySelector('.action');
+    const text = input.value.trim();
+    if (text !== '') {
+        add(text);
+        input.value = '';
+        input.focus();
+    }
+});
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for this project, " `On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.` ", please kindly ensure that, On page load(or reload), the task array is rendered dynamically to the DOM. The tasks should appear in order of the index values for each task.